### ISSUE TITLE
Suffocation debuff too short with too long delay before it applied again.

### ISSUE THREAD

**Issue by Xerkus** [RavienGaming/Reforged-Eden-2#163]
### Reforged Eden 2 Build Number

13

### Game mode

Survival

### Tested in vanilla

Issue does not happen in vanilla

### Description of issue

Suffocation debuff is applied for 1 second with very long delay before it is applied again. Slow regeneration with alpha implant can keep person alive indefinitely. Debuff is applied more often while jumping and running but standing still I abandoned timing attempt after 10 minutes. 

Same effect in space, with or without helmet and in a new game.

**Comment by VermillionX** [comment]
I have no idea what you're trying to say

**Comment by Xerkus** [comment]
- Start new game
- Go into space in suit
- Remove helmet to start suffocating or wait for suit oxygen to run out
- Observe not getting suffocating debuff
- Observe not dying from lack of oxygen

**Comment by Xerkus** [comment]
I think problem is not that debuf is not applied but conditions for its removal are broken leading to status effect getting immediately removed before UI had time to react.

I observed screen sometimes flashing with red suffocation overlay with no status icon appearing which supports my conclusion.

Also, I managed to catch suffocation debuff ingame name, it is simply "No Oxygen"

**Comment by VermillionX** [comment]
RE2 and Vanilla NoOxygen status effect are almost identical; differing only in how much health is taken off. If it's cancelling itself too early in RE2, then it's doing the same in vanilla.

**Comment by Xerkus** [comment]
I did a series of tests again:
New RE2 start on Omicron.  I removed helmet and NoOxygen applied properly. Then I did as in all previous tests: `gm on` to fly to space in the suit. In space `gm off` then restart the game. After loading into save I remove helmet and see that No Oxygen is not applied. Upon return to Omicron issue persisted as well.

Same test in RE1: issue not observed.
Same test in vanilla: issue not observed.

Another RE2 test: in a new game use `sbp` to spawn ship and get to space that way. Result: No Oxygen applied normally. Looks like using `gm` permanently affects character in this aspect but for some reason only in RE2.
